content/references/protocol-reference/transactions/transaction-types/clawback.md

```diff
@@ -0,0 +1,64 @@
+---
+html: clawback.html
+parent: transaction-types.html
+blurb: Claw back an issued currency transfer between accounts in a trust set.
```

Comment:
```suggestion
blurb: Claw back tokens you've issued.
```

Reply:
Thank you, Rome, for your thorough review.